packages/affiliates/gas-rebate/VoterGasRebateV2.ts

```diff
@@ -44,8 +44,17 @@ export async function run(): Promise<void> {
 
   // Fetch all commit and reveal events.
   const voting = await ethers.getContractAt("VotingV2", await getAddress("VotingV2", 1));
-  const commitEvents = await voting.queryFilter(voting.filters.VoteCommitted(), fromBlock, toBlock);
-  const revealEvents = await voting.queryFilter(voting.filters.VoteRevealed(), fromBlock, toBlock);
+
+  const searchConfig = {
+    fromBlock,
+    toBlock,
+    maxBlockLookBack: 20000,
+  };
+
+  // Find resolved events
+  const commitEvents = await paginatedEventQuery<any>(voting, voting.filters.VoteCommitted(), searchConfig);
```

Comment:
I had issues running the script because the queryFilters were returning more than `10_000` results so I switched to this `paginatedEventQuery`